### DISCUSSION TITLE
Doesn't Circle set filter?

### DISCUSSION THREAD

**Discussion by wuxingping** [fabricjs/fabric.js#9003] [General]
I set the option {filter: 20 } for Circle, like path and Image, but it doesn't work. why?

**Comment by ShaMan123** [comment]
MUST reproduce always

**Reply by wuxingping** [reply]
https://codesandbox.io/s/fabric-app-forked-55gqxf?file=/src/ManyLayouts.tsx

as you see, I have set the prop of filter for Circle and Path, but it didn't work.
` const circle = new Circle({
      left: 100,
      radius: 50,
      fill: "red",
      filter: 20
    });`

![image](https://github.com/fabricjs/fabric.js/assets/41353266/e3c8c8a1-a066-4e1c-a5f1-7e52bc95c509)
